<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>typingTestWindow</class>
 <widget class="QMainWindow" name="typingTestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Back To Home</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="timerLabel">
        <property name="styleSheet">
         <string notr="true">font-size: 18px;
 font-weight: bold; 
color: blue;</string>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="textLabel">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Text will be appear here....</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="typedTextLabel">
          <property name="styleSheet">
           <string notr="true">background: white; border: 1px solid black; min-height: 40px;</string>
          </property>
          <property name="text">
           <string>&quot;&quot;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="keyboardWidget">
      <item row="1" column="3">
       <widget class="QPushButton" name="button_E">
        <property name="text">
         <string>E</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="button_4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="4" column="12" colspan="2">
       <widget class="QPushButton" name="button_RShift">
        <property name="text">
         <string>Shift</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="button_F">
        <property name="text">
         <string>F</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="button_S">
        <property name="text">
         <string>S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="button_7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QPushButton" name="button_0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="13">
       <widget class="QPushButton" name="button_Backspace">
        <property name="text">
         <string>Backspace</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="button_CapsLock">
        <property name="text">
         <string>CapsLock</string>
        </property>
       </widget>
      </item>
      <item row="2" column="11">
       <widget class="QPushButton" name="button_Apostrophe">
        <property name="text">
         <string>'</string>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QPushButton" name="button_L">
        <property name="text">
         <string>L</string>
        </property>
       </widget>
      </item>
      <item row="4" column="11">
       <widget class="QPushButton" name="button_Slash">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="button_U">
        <property name="text">
         <string>U</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QPushButton" name="button_K">
        <property name="text">
         <string>K</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QPushButton" name="button_I">
        <property name="text">
         <string>I</string>
        </property>
       </widget>
      </item>
      <item row="4" column="8">
       <widget class="QPushButton" name="button_M">
        <property name="text">
         <string>M</string>
        </property>
       </widget>
      </item>
      <item row="1" column="11">
       <widget class="QPushButton" name="button_LBracket">
        <property name="text">
         <string>[</string>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QPushButton" name="button_N">
        <property name="text">
         <string>N</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="button_A">
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="button_3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QPushButton" name="button_Dash">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="10">
       <widget class="QPushButton" name="button_P">
        <property name="text">
         <string>P</string>
        </property>
       </widget>
      </item>
      <item row="2" column="10">
       <widget class="QPushButton" name="button_Semicolon">
        <property name="text">
         <string>;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="button_D">
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QPushButton" name="button_9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="4" column="9">
       <widget class="QPushButton" name="button_Comma">
        <property name="text">
         <string>,</string>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QPushButton" name="button_Equals">
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="button_H">
        <property name="text">
         <string>H</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="button_Tab">
        <property name="text">
         <string>Tab</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="button_2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="button_W">
        <property name="text">
         <string>W</string>
        </property>
       </widget>
      </item>
      <item row="1" column="12">
       <widget class="QPushButton" name="button_RBracket">
        <property name="text">
         <string>]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QPushButton" name="button_J">
        <property name="text">
         <string>J</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QPushButton" name="button_O">
        <property name="text">
         <string>O</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="button_5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="button_R">
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="4" column="10">
       <widget class="QPushButton" name="button_Period">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="button_Backtick">
        <property name="text">
         <string>`</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="button_T">
        <property name="text">
         <string>T</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button_Q">
        <property name="text">
         <string>Q</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="button_G">
        <property name="text">
         <string>G</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4" colspan="5">
       <widget class="QPushButton" name="button_Space">
        <property name="text">
         <string>Space</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="button_1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="button_6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QPushButton" name="button_8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="button_Y">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="13">
       <widget class="QPushButton" name="button_Backslash">
        <property name="text">
         <string>\</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QPushButton" name="button_B">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="button_V">
        <property name="text">
         <string>V</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="button_C">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="button_X">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="button_Z">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="button_LShift">
        <property name="text">
         <string>Shift</string>
        </property>
       </widget>
      </item>
      <item row="2" column="12" colspan="2">
       <widget class="QPushButton" name="button_Enter">
        <property name="text">
         <string>Enter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
